### DISCUSSION TITLE
Analysis of 022, 022b, 022c, 022d + Next coupled simulation

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#172] [TF5: Coupled SE/MOM6]
Today at the co-chair meeting we discussed the latest set of runs: 022, 022b, 022c, 022d, 024. 

![Screen Shot 2022-08-30 at 2 11 31 PM](https://user-images.githubusercontent.com/9723220/187533814-e0cd3de8-7de7-4fe6-aefc-84c0c16b2768.png)

![](https://user-images.githubusercontent.com/56696811/187463068-3ceb74b2-f236-4b13-8637-d28ed06f51d7.png)

[Full presentation](https://docs.google.com/presentation/d/1_Yf68tNZbv8R960Lwk6_mFB3HzfxJNjO/edit?usp=sharing&ouid=110527899490311587217&rtpof=true&sd=true) is on the cam7 drive

We decided we would start two new coupled simulation:
25a: same a 24 + spunup ice (from 016 at yr 58) + tuning
25b: same a 24 + spunup ice/ocn (from 016 at yr 58) + tuning

We need to provide tuning recommendations for that run. 
- do we want to turn on trapezoidal rules or not ? See 24 (trapezoidal rule off) and 23 (trapezoidal rule on) in [F cases](https://github.com/NCAR/amwg_dev/issues/163) and in [B cases ](https://github.com/NCAR/amwg_dev/issues/161)
- examine clubb_l_call_pdf_closure_twice impact in [F case](https://github.com/NCAR/amwg_dev/issues/171)
- we need to adjust club tuning to increase the cloud. Do we want to adjust c8, gamma-coef, c14 ?

- 024 tuning:
```
clubb_gamma_coef   =  0.28
clubb_c14          = 2.2D0
clubb_l_trapezoidal_rule_zm          =  .false.
clubb_l_trapezoidal_rule_zt          =  .false. 
clubb_c8		=  4.2   
```
- 023 tuning:
```
clubb_gamma_coef   =  0.28
clubb_c14          = 2.2D0
clubb_l_trapezoidal_rule_zm          =  .true.
clubb_l_trapezoidal_rule_zt          =  .true. 
clubb_c8		=  4.2   
```
- 009 tuning:
```
clubb_gamma_coef     =  0.270   
clubb_c14            =  1.6D0  
clubb_l_trapezoidal_rule_zm          =  .false.
clubb_l_trapezoidal_rule_zt          =  .false. 
clubb_c8		=  4.2   
```

If we want to explore the impact of ``clubb_c8``, we may look at [19b](https://github.com/NCAR/amwg_dev/issues/149) 
```
clubb_c8               =  4.2 -> 4.6 
```
resulted in a decrease of restom of about -1.7 W/m2 and an increase of SWCF of -2.2W/m2 (more cloud). 

I can do a F case before doing the full coupled simulation 

@adamrher, @JulioTBacmeister

**Comment by adamrher** [comment]
I think that the evidence suggests we should keep the trapezoidal rules on (see [here](https://github.com/NCAR/amwg_dev/issues/171#issuecomment-1232141253)). That means we have to keep the l_call_pdf_closure_twice=T, too. That basically gives us the settings of run 23, whose clouds are too thin.

I'm not quite sure how big of a hammer to apply to clubb tunings for a modified run 23. run 23 has a +1.5 W/m2 RESTOM, and a -42W/m2 SWCF. Here are some options that seem reasonable (all mods will thicken clouds):

1. clubb_gamma = 0.280->0.270
2. clubb_gamma = 0.280->0.270, clubb_c14=2.2->1.6 
3. clubb_c8 = 4.2->4.4
4. clubb_c8 = 4.2->4.4, clubb_c14=2.2->1.6
5. clubb_c8 = 4.2->4.6

@swrneale @andrewgettelman

**Comment by andrewgettelman** [comment]
I'm not sure I have a lot of insight here. I guess one question is Which clouds are the problem. clubb_gamma hits the subtropical stratus and stratocumulus regions, while c8 is more storm tracks in the NH. So they are orthogonal. Why not combine clubb gamma and c8? I'm not sure what c14 is doing.

I'm looking at some older analysis from CAM5: https://agupubs-onlinelibrary-wiley-com.cuucar.idm.oclc.org/doi/10.1002/2014MS000405

**Comment by cecilehannay** [comment]
Justin plot the [tables 19b vs 17 ](https://webext.cgd.ucar.edu/B1850MOM/b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.019b/atm/b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.019b_25_35_vs_b.cesm3_cam058_mom_e.B1850MOM.ne30_L32_t061.cam6_cice5.017_25_35/html_table/amwg_table_Case%20Comparison.html) to look at the impact of ```clubb_c8 = 4.2->4.6```

If the numbers are correct, the swcf increases by -2.25W/m2 (more cloud). The restom decreases by -0.4 W/m2. (Here, I was expecting the change in restom to be more dramatic.).   

Adam suggested above that we start from 23. Do we want to do a f case first or jump into the b case right away?

**Reply by dan800** [reply]
_Here, I was expecting the change in restom to be more dramatic._
I think that's because the surface has cooled by almost 1K (more so on land), so while the clouds are reflecting more shortwave radiation, the longwave upwelling flux has also dropped.

**Reply by adamrher** [reply]
it also thickens the clouds w.r.t longwave --- LWCF increases by a W/m2 and so buffers the SWCF signal.

**Reply by dan800** [reply]
I can only see global mean tables, but the trends on TS are alarming (-0.1 K/y in 17). Did these runs start from the same i.c.? Trying to reconcile that while 17 has a bigger cooling trend at year 25 it was warmer by almost 3K.
17: TS 292.626 + -0.118 t
19b: TS 289.896 + -0.056 t

**Reply by adamrher** [reply]
Julio's got some line plots here: https://github.com/NCAR/amwg_dev/issues/149#issuecomment-1202763887. Looks to me like they are all starting from the same state.

**Reply by dan800** [reply]
Thanks @adamrher Looks like this metric isn't useful on a 12 year record.

**Comment by adamrher** [comment]
@andrewgettelman I see the Guo plots show a very different spatial patterns of sensitiviy of c8 and clubb_gamma. However, our experiments indicate that they have ~identical spatial patterns -- they primarily target marine Sc region, but in addition you see diffuse impacts extending to the storm tracks. See here https://github.com/NCAR/amwg_dev/issues/149#issuecomment-1228983320.

I'm paraphrasing Vince, but he tends to like targeting c8 over gamma, since it is a more physical pathway to impacting skewness (it is the damping coefficient for the w'3 eqn). Whereas clubb_gamma is more of a shape parameter for the pdf. 

We are also almost saturated w.r.t clubb_gamma -- anything lower than 0.270 is the red zone / uncharted territory. (I'll note Guo took liberties to test an extremely large range of values that are not physically plausible).

Cceile is reporting that in L32, c8=4.2->4.6 

> If the numbers are correct, the swcf increases by -2.25W/m2 (more cloud). The restom decreases by -0.4 W/m2. (Here, I was expecting the change in restom to be more dramatic.).

That pertains to my option (5). I kind of feel like it would be useful to do two runs to really capture the c8 sensitivity at L58. Big c8 change (option (5)) and mild c8 change (option (3)). There seems to be large offsets of the SWCF by LWCF, so would be good to see the degree they cancel in these two expts.

**Reply by cecilehannay** [reply]
Sounds goo. 

I will do 2 F runs with 
- mild change in c8 ``clubb_c8 = 4.2->4.4``
- big change in c8. ``clubb_c8 = 4.2->4.6``

Then we can decide tomorrow or thrusday how to set the b cases we talked about today.

**Reply by cecilehannay** [reply]
Links to:
[run with mild change in c8](https://github.com/NCAR/amwg_dev/issues/173) ``clubb_c8 = 4.2->4.4``  
[run with big change in c8.](https://github.com/NCAR/amwg_dev/issues/173) ``clubb_c8 = 4.2->4.6``

**Comment by cecilehannay** [comment]
@justin-richling plotted the ADF for the two flavors of clubb_c8

- clubb_c8 = 4.2->4.4 
[f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.c8_mild.023 vs f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.023](https://webext.cgd.ucar.edu/FWscHIST/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.023/atm/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.c8_mild.023_1980_1985_vs_f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.023_1980_1985/)

- clubb_c8 = 4.2->4.6
[f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.c8_big.023 vs f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.023](https://webext.cgd.ucar.edu/FWscHIST/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.023/atm/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.c8_big.023_1980_1985_vs_f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.023_1980_1985/)

**Reply by adamrher** [reply]
recording the delta SWCF and delta RESTOM here:

(these are from the ADF tables, which I think are wrong)
- clubb_c8 = 4.2->4.4 (dRESTOM = -1.03 W/m2, dSWCF = -1.16 W/m2)
- clubb_c8 = 4.2->4.6 (dRESTOM = -1.69 W/m2, dSWCF = -2.16 W/m2)

(these are from ADF lat-lon plots, which I think are correct -- had to reconstruct RESTOM from FLNT/FSNT)
- clubb_c8 = 4.2->4.4 (dRESTOM = -0.86 W/m2, dSWCF =  -1.0 W/m2)
- clubb_c8 = 4.2->4.6 (dRESTOM =  -1.82 W/m2, dSWCF = -2.25 W/m2)

**Reply by adamrher** [reply]
And here's SWCF comparison against run23, and CERES:

[temp_SWCF.c8mild.pdf](https://github.com/NCAR/amwg_dev/files/9465225/temp_SWCF.c8mild.pdf)
[temp_SWCF.c8big.pdf](https://github.com/NCAR/amwg_dev/files/9465505/temp_SWCF.c8big.pdf)

Shall we go big? It's clearly a more realistic SWCF field. I'm a little worried about the large dRESTOM though. The b-case version of run 23 (https://github.com/NCAR/amwg_dev/issues/158) has a RESTOM of +1.53. I mean it's great that we can get RESTOM close to zero ... but so far all our successful cases (i.e., convection in lab sea) tend to have RESTOM on the positive side ... 19b is the L32 version of when we increased c8=4.2->4.6, and the RESTOM was close to zero, but the lab sea convection was extremely weak.

I'm almost thinking of recommending c8=4.2->4.5 for the b-case. Thoughts anyone?

**Reply by cecilehannay** [reply]
I think we should go big or if we are concerned about the restom to go at least c8=4.2->4.5.
The change in restom in the f case can be quite different from the change in restom in the b case.

**Reply by cecilehannay** [reply]
That said, I am a bit concerned about the numbers in the ADF tables that don't match the numbers in the ADF lat-lon plots. We should try to see why this is the case.

**Reply by adamrher** [reply]
Oh, yeah I see that now. Comparing to my SWCF plots, I think the global numbers in ADF plots are more accurate. 

For 08mild, years 1980-1985:

ADF Table = -44 W/m2
ADF Plot = -43.23 W/m2
My Plot = -43.2243 W/m2

Let me try to update the dRESTOM, dSWCF numbers with the plot values instead of the table.

**Reply by andrewgettelman** [reply]
I think we go big. Most of the impact of this is tropical, and not related to the high latitudes. Will cooling the tropics freeze the Lab sea? It's more correct. If if craters, we back off? 

As Cecile notes RESTOM in F-cases doesn't directly translate (often lower I think).

**Reply by adamrher** [reply]
here's the updated stats,

> clubb_c8 = 4.2->4.4 (dRESTOM = -0.86 W/m2, dSWCF = -1.0 W/m2)
clubb_c8 = 4.2->4.6 (dRESTOM = -1.82 W/m2, dSWCF = -2.25 W/m2)

So the dRESTOM is bit larger than we originally estimated. These numbers may not map onto what happens in a b-case, but we don't know which way the b-case is going to go, (bigger or smaller?). I'm going to advocate for big'ish, c8=4.2->4.5.

**Reply by cecilehannay** [reply]
The change in restom is often much lower in the b case than in the f case.

**Reply by dan800** [reply]
OK, I am guessing this is obvious to you all, but can I confirm that that is simply because if you are looking at, for example, years 20 to 32, the surface ocean has warmed or cooled a lot to bring the atmosphere into equilibrium (i.e., a rapid adjustment of the shallow ocean)? In contrast, the f case is going to stay much closer to the instantaneous change in RF. All I think you can tell from a b-case RESTOM is how far out of equilibrium the atmosphere is from the spun up model ocean state and an indication of how long it will take for the whole ocean to warm or cool. In other words, the degree to which you've picked settings in physics that match the RF forcing at the surface that the spun up ocean has experienced.

**Reply by adamrher** [reply]
that's a convincing explanation for a smaller RESTOM response in a b-case, once the surface ocean has had time to respond.

I believe that in most experiments (unless otherwise stated), the b-cases are initialized from the Levitus World Atlas observational product. So it hasn't experienced much spin-up --its state is the target observational state of the ocean. (this gives 3.6K global ocean T ... note that CESM2 pi control was initialized from this same state, but over the course of 1500 years of so, the ocean warmed from 3.6K->4.2K ... so some drift is apparently acceptable).

**Reply by adamrher** [reply]
Here's RESTOM for 12 years:

![temp_tseries_RESTOM](https://user-images.githubusercontent.com/29961476/187956135-ec0f8e97-d613-44af-8ea9-739919583ddb.png)

c8=4.2->4.4, dRESTOM = -1.04 W/m2
c8=4.2->4.6, dRESTOM = -1.93 W/m2

I think we have enough information to make the recommendation for the b-case. Go big or go big'ish? I'm still for big'ish (4.2->4.5) ... mainly guided by fear -- Ill be devastated if lab sea convection shuts off due to c8 (which did occurr in L32).

**Comment by cecilehannay** [comment]
I can start 025a and 025b with 
c8 = 4.2->4.5
with 025a = ice spunup and 025b = ocn/ice spunup 

I think the SWCF will still be anemic especially over stratocumulus regions. We can do more adjustment in 026 - maybe with gamma_coef. 

I agree with @adamrher that baby steps might be good for now instead of going full speed and freezing the lab sea.

**Reply by adamrher** [reply]
I agree, but open to criticism. I don't think we need to get the clouds right this time around, we just need to get RESTOM a bit closer to reality while we try to keep the lab sea ice free. And I think we need to be sure that we begin on the right side of the lab sea attractor ... if we begin on the wrong side, there's nothing we can do. This is also informing my preference for big'ish over big.

**Reply by cecilehannay** [reply]
We are both on the same page. I will start 025a and 025b.

**Reply by dan800** [reply]
@adamrher  _we just need to get RESTOM a bit closer to reality_
Based on the '80s and '90s real atmosphere warming rate what do want in terms of a RESTOM? Something between 0.5 and 1 W/m2? (e.g., https://www.nature.com/articles/nclimate3043). Do all these runs have El Chichon going off near the beginning?

**Reply by adamrher** [reply]
I'm interested in the change in SWCF from f.023 and the other two f-case runs. I'm looking for a delta to bring our b.023 run down closer to neutral RESTOM (it's about +1.5 W/m2). That's a good point that el chichon is probably built into the forcing for these f-case runs though ... that certainly contributes to it being an imperfect analogy for parameterization sensitivity in b-cases.

**Reply by adamrher** [reply]
we should probably be running an F1850 type f-case...

**Reply by dan800** [reply]
Would it be better to use the 2010 climo compset for F tests? At least in the present day F tests I don't think we want a neutral RESTOM (though +1.5 W/m2 is likely too much).
This was in reply to your prior comment. An F1850 could also work.

**Reply by adamrher** [reply]
I think F1850 would be the best option for informing parameters in B1850. But recall that Julio showed in his last slide, last friday, that using FHIST to test sensitivity to changing physics in B1850 was fairly accurate ... even though the absolute RESTOM in FHIST was very different than that in B1850.

**Reply by swrneale** [reply]
This is the premise we have worked from for years. F-cases have an infinite source of heat for sigma*T^4 emission that is unencumbered by overlying clouds. The distribution of SST errors in B cases and the strong non-linear dependence of surface fluxes makes it very challenging to get a similar RESTOM. Therefore we tune in coupled mode where it really matters and F-cases are what they are as it matters less.

**Reply by dan800** [reply]
I agree that mean cloud biases make for inconsistencies between ocean regions that in reality radiate to space but in the model are blocked by clouds (and vice-versa). On average, though I wonder if you have a reasonable total cloud if it's a wash. I do think there's a lot to be learned from looking at the instanteous RF from a clubb or other physics knob twiddle (and it's a lot cheaper). You'd hope our physics was good enough that a model initialized with something close to observed SSTs/sea ice and consistent GHG loading that was originally balanced would not drift to a drastically different mean TS. 

But perhaps it does due to the non-linear effect you bring up. From your experience, when including these non-linear effects is the b-case always warmer of cooler than a f-compset? In other words, if you initialized the interactive ocean to observations and allowed the clouds to interact with the surface temperature which way would the model trend in the near term?

**Comment by swrneale** [comment]
I'm wondering what the exact correlation is between RESTOM biases and north Atlantic SST evolution is, i.e., is it clear that we should invest a lot of our time in global tuning in order to address the lab sea freezing in repeated initial experiments, or more time in manipulating the ICs somehow?

**Reply by cecilehannay** [reply]
@swrneale: this is partly why we are trying to do in 25a and 25b (at least partly).
My preference would be to start from the end of 22d or some state that clearly don't freeze. We would have a better chance to keep the lab sea from freezing. 
For me the control 016 is kinda of borderline.  
But the oceanographers were a bit concerned because 22d had drifted quite a bit. 
I think Frank and Gustavo are going to look into this.

Right now, we are trying to get back some clouds in the model with c8.

**Comment by cecilehannay** [comment]
Justin posted the timeseries for the beginning of the simulations 25a and 25b just to make sure there no showstopper so far:
![188201844-36e6d26f-af0f-4f66-b511-9ecdf6b8b0f9](https://user-images.githubusercontent.com/9723220/188203665-a678c5f2-c3ae-4640-886e-8164ad97bcdd.png)
